lighting/views/views.xml

```diff
@@ -223,7 +224,10 @@
                                     widget="many2many_tags"
                                     options="{'color_field': 'color'}"
                                 />
-                                <field name="is_lamp_included" />
+                                <field
+                                    name="is_lamp_included"
+                                    attrs="{'column_invisible': [('parent.line_common_invisible_fields', 'in', ('is_lamp_included',))]}"
+                                />
```

Comment:
What would happen if a source has 3 lines and one line has 'lamp included' filled in but it is not a relevant field, and another line has 'lamp included' filled in and it is relevant? The column would then be displayed, and the column would show the 'is_lamp_included' activated for the line where it is not relevant, right?